src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -2355,6 +2538,30 @@ GroupedByDisjointPartitionColumn(List *tableNodeList, MultiExtendedOp *opNode)
 }
 
 
+static SortGroupClause *
+createGroupByClause(TargetEntry *targetEntry, Index nextSortGroupRefIndex)
```

Comment:
Missing comments, starts with lower case.

Reply:
Maybe rename to `CreateSortGroupClause()`

Reply:
Can we do this on the caller side, it puts more functionality than the function name sounds
```
targetEntry->ressortgroupref = nextSortGroupRefIndex;
```

Reply:
makes sense fixed